.github/workflows/ci-ubuntu.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       matrix:
         ocaml_version: ["4.14"]
-        os: ["ubuntu-latest"]
-        gcc_version: ["11"]
+        os: ["ubuntu-22.04", "ubuntu-latest", "ubuntu-24.04-arm"]
```

Comment:
I suggest we drop ubuntu-arm, as nobody use it

Reply:
See [2716a93](https://github.com/o1-labs/orocksdb/pull/3/commits/2716a93a1ff86828a30e841ec6cde96d2c33424b)